src/cmds/restic/cmd_backup.go

```diff
@@ -293,6 +303,10 @@ func readLinesFromFile(filename string) ([]string, error) {
 }
 
 func runBackup(opts BackupOptions, gopts GlobalOptions, args []string) error {
+	if opts.FilesFrom == "-" && gopts.password == "" && gopts.PasswordFile == "" {
```

Comment:
Does the following error should only raise when files to back up come from stdin (-)? I am very new to this code and probably I am missing things, but... I ask just in case. Thx!

Reply:
Try it with the latest version, we've recently changed this part of the code. if you discover anything odd with it, please create a new issue! Comments to old pull requests tend to get lost :)

Reply:
Oh, thanks, you're right. I'm rather new to this workflow. Thanks for pointing that :)